apps/tactics-finder/worker.py

```
@@ -3,27 +3,30 @@
 import requests
 import time
 import json
-from dotenv import load_dotenv
-from typing import Optional, Tuple, List
-from multiprocessing import Pool, cpu_count
 import uuid
 import io
+import hashlib
+from dotenv import load_dotenv
+from typing import List
```

Ruff (0.14.5)

`typing.List` is deprecated, use `list` instead

(UP035)